docs/tutorial.md

```diff
@@ -349,7 +349,7 @@ This tutorial has focused on Ruff's command-line interface, but Ruff can also be
 ```yaml
 - repo: https://github.com/astral-sh/ruff-pre-commit
   # Ruff version.
-  rev: v0.1.4
+  rev: v0.8.2
   hooks:
```

Comment:
We can add this file to https://github.com/astral-sh/ruff/blob/3d9ac535e92eac324776f94c8760b7220f45e624/pyproject.toml#L106-L113 so that it gets updated automatically on every release.